packages/documentation/docs/controls/_breadcrumb_styleguide.md

```diff
@@ -1,97 +1,44 @@
 Breadcrumb navigation is a UI control that allows users to track their location within an application and easily navigate to previous or child pages.
+Breadcrumbs make the structure of your application transparent to the user. They offer means to navigate through the application. We typically use breadcrumbs when your applications have a deep hierarchy of pages or content. This can help users understand where they are within the application, and make it easier for them to navigate back to previous pages. As a general rule, we don't use breadcrumbs for an information architecture of two levels or less. We don't use breadcrumbs as replacement for a main navigation. If the information structure is extremely complex, you can consider using a tree instead of a breadcrumb.
```

Comment:
```suggestion
Breadcrumbs make the structure of your application transparent to the user. They offer means to navigate through the application. We typically use breadcrumbs in applications that have a deep hierarchy of pages or content. This can help users understand where they currently are within the application, and make it easier for them to quickly navigate to pages further up in the navigation tree. As a general rule, we don't use breadcrumbs for an information architecture of two levels or less. We don't use breadcrumbs as replacement for a main navigation. If the information structure is extremely complex, you can consider using a tree instead of a breadcrumb.
```